### PR TITLE
Support Ceph RBD storage plugin v0.1.0

### PR DESCRIPTION
**Pull request by niusmallnan**
https://github.com/rancher/rancher/issues/8722

### CODE REVIEW

File: infra-templates/rbd/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.catalog:
+  name: "Rancher RBD"
+  version: "0.1.0"
+  description: |
+    Docker volume plugin for Ceph RBD
+  minimum_rancher_version: v1.6.6-rc1
```
Comment (deniseschannon) [inline infra-templates/rbd/0/rancher-compose.yml]: can we update this to v1.6.8-rc1?

Comment (niusmallnan) [reply infra-templates/rbd/0/rancher-compose.yml]: done

**Comment by yasker** [comment]
@niusmallnan 

Can you rebase on `v1.6-cn-development` branch? It's created to expedite the process for China team catalog to be merged to `v1.6-release`. We don't need to wait for `1.6-development` to be merged now.

**Comment by niusmallnan** [comment]
@yasker @deniseschannon 
rebased on `v1.6-cn-development` branch

File: infra-templates/rbd/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.catalog:
+  name: "Rancher RBD"
+  version: "0.1.0"
```
Comment (deniseschannon) [inline infra-templates/rbd/0/rancher-compose.yml:3]: I wonder if we should match the version to the image version. it makes it easier to distinguish. Sorry, thought about it when reviewing the other PR.

**Comment by Tracyhuahua2020** [comment]
This pr had been tested and passed.
The test case as below https://github.com/Tracy1994/rancher-test_case/blob/master/rbd_testcase.xlsx.
My suggestion is we should remind our user this ctalog only available from aliyun VPC  network and rancher-server version should be 1.6.8+.
There is a bug about storage should be fix. https://github.com/rancher/rancher/issues/9834

**Pull request merged**